recipes/aocl-blas/recipe.yaml

```
@@ -0,0 +1,58 @@
+schema_version: 1
+
+context:
+  version: 5.1
+
+package:
+  name: aocl-blas
+  version: ${{ version }}
+
+source:
+    url: https://github.com/amd/blis/archive/refs/tags/${{ version }}.tar.gz
+    sha256: 4ab210cea8753f4be9646a3ad8e6b42c7d19380084a66312497c97278b8c76a4
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - ${{ stdlib("c") }}
+    - ${{ compiler("c") }}
+    - ${{ compiler("cxx") }}
+    - if: not win
+      then: ${{ compiler("fortran") }}
+    - if: not linux
+      then: llvm-openmp
+    - if: linux
+      then: libgomp
+    - if: not win
+      then: make
+    - if: win
+      then: ninja
+    - cmake
```

```suggestion
    - if: not win
      then: ${{ compiler("fortran") }}
    - if: not linux
      then: llvm-openmp
    - if: linux
      then: libgomp
    - ninja
    - cmake
```
Let's just use Ninja on all platforms.

---

Done!